### PR TITLE
game.controller.ps: update layout.xml to the dualshock controller

### PR DESCRIPTION
**Pull request by lrusak**
EDIT: I mean the dual shock controller 

This updates the controller mapping to be similar to this controller
Dual Analog Controller

### CODE REVIEW

File: addons/game.controller.ps/layout.xml [hunk 1]
```diff
@@ -7,19 +7,24 @@
 		<button name="triangle" type="digital" label="30004"/>
 		<button name="start" type="digital" label="30005"/>
 		<button name="select" type="digital" label="30006"/>
-		<button name="up" type="digital" label="30007" direction="up"/>
-		<button name="down" type="digital" label="30008" direction="down"/>
-		<button name="right" type="digital" label="30009" direction="right"/>
-		<button name="left" type="digital" label="30010" direction="left"/>
-		<button name="l3" type="digital" label="30015"/>
-		<button name="r3" type="digital" label="30016"/>
+		<button name="guide" type="digital" label="30007"
```
Comment (garbear) [inline addons/game.controller.ps/layout.xml]: the dualshock doesn't have a guide button, right?

Comment (lrusak) [inline addons/game.controller.ps/layout.xml]: It has the "analog" button, not sure if this actually does anything though

Comment (garbear) [inline addons/game.controller.ps/layout.xml]: I think analog is a hardware button that modifies the bahavior of the controller. Not sure though, we should look through the libretro.cpp file of our psx emulators to see if the guide button maps to anything

Comment (lrusak) [inline addons/game.controller.ps/layout.xml]: Doesn't look like it, https://github.com/libretro/beetle-psx-libretro/blob/master/libretro.cpp#L2757

I can remove it

**Comment by lrusak** [comment]
Remove the guide button

**Comment by garbear** [comment]
cool thanks

**Pull request merged**